layouts/partials/download-list.hbs

```diff
@@ -1,5 +1,6 @@
 <section>
   <ul>
+    <li><a href="https://docs.microsoft.com/windows/dev-environment/javascript/nodejs-on-wsl">{{site.download.install-on-wsl.text}}</a></li>
```

Comment:
looks like this is still top of the list and needs to be moved to the bottom.

Regarding the order, all the other links are to resources provided directly from the Node.js project. Considering that this is an off site resource I think it only make sense to include it as the last link

to @nschonni's question regarding why this isn't in the other package manager instructions I do see this as distinct as it is about setting up on a platform, not a specific way to install Node.js, you still need to manage it on WSL, in which case nvm may be the right choice. It would likely be good to do a follow up to improve some other documentation related to this, but I think that have a clear pointer to high quliaty maintained instructions on using Node.js with WSL is a net positive